spring-integration-etcd/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+etcd:
+  image: microbox/etcd
+  ports:
+    - "4001:4001"
+    - "7001:7001"
+  command: "--name spring-cloud-cluster-etcd --data-dir=/data"
```

Comment:
How are you going to support users that want to test it? Docker compose is the best way to do that IMO.